Classes/Core/Testbase.php

```diff
@@ -701,6 +701,7 @@ private function truncateAllTablesForMysql(): void
             if ($isChanged) {
                 $tableName = $tableData['table_name'];
                 $connection->truncate($tableName);
+                self::resetTableSequences($connection, $tableName);
```

Comment:
We actually dropped that by intention: This resetTableSequences() call is only needed for postgres, it's obsolete with mysql.

Reply:
Ah, okay. Thanks for the explanation!

Should we make this PostgreSQL-only then (in a separate PR)?

Reply:
Ah, it already is.

Reply:
Unsure. There's a switch for sqlite in there, too. It's just that it was clearly obsolete for mysql/maria.